.github/workflows/test.yml

```diff
@@ -17,24 +17,35 @@ jobs:
       matrix:
         include:
           - ruby: truffleruby
+            rubygems: default
           - ruby: truffleruby+graalvm
+            rubygems: default
           - ruby: jruby
+            rubygems: default
           - ruby: "2.5"
+            rubygems: "2.7.11"
             rake_task: "rubocop test"
           - ruby: "2.6"
+            rubygems: default
           - ruby: "2.7"
+            rubygems: default
           - ruby: "3.0"
+            rubygems: default
           - ruby: "3.1"
+            rubygems: default
             coverage: "yes"
           - ruby: "3.2"
+            rubygems: default
     env:
+      RUBYOPT: --enable-frozen-string-literal
```

Comment:
Would it be better to add the frozen string literal comment to the file(s)? That way Ruby can use frozen strings even at runtime.

Reply:
It would be better for the consumers of the gem. It changes nothing for checking the compatibility of running with strings frozen.

I'd prefer to separate it out.